application/src/main/java/org/togetherjava/tjbot/commands/tags/TagsCommand.java

```diff
@@ -42,8 +45,17 @@ public TagsCommand(TagSystem tagSystem) {
     @Override
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
         // TODO A list might be better than comma separated, which is hard to read
-        event.replyEmbeds(MessageUtils.generateEmbed("All available tags",
-                String.join(", ", tagSystem.getAllIds()), event.getUser(), TagSystem.AMBIENT_COLOR))
+        ArrayList<String> list = new ArrayList<>(tagSystem.getAllIds());
```

Comment:
This list variable should use the interface List and not ArrayList

Reply:
The `List` interface does not have a `subList` method

Reply:
I'm pretty sure it does
https://docs.oracle.com/javase/8/docs/api/java/util/List.html#subList-int-int-